domain-models-runtime/src/main/java/org/folio/rest/persist/Criteria/Criterion.java

```diff
@@ -252,7 +269,12 @@ public String selects2String() {
     return sb.toString();
   }
 
-  public String from2String() {
+  /**
+   * @return as clause for SELECT of empty string if none
+   * @deprecated not used by RMB itself anymore and should not be publicly used
+   */
+  @Deprecated
+  public String from2String1() {
```

Comment:
Either keep the name (don't rename to from2Strin1) and make it deprecated, or completely remove it.
I suggest to remove it because this is part of a breaking change and not used: https://github.com/search?q=org%3Afolio-org+from2String&type=Code

Reply:
Also here.